src/state/slices/opportunitiesSlice/opportunitiesSlice.ts

```diff
@@ -113,13 +115,20 @@ export const opportunitiesApi = createApi({
           // TODO: This commit authors LP slice population only - for Fox staking we will want to assign this to a variable and actually use the data
           // The reason for that is for EVM chains LPs, we just need to await this promise resolution - if this resolves, it means we have portfolio data
           // If this throws, the RTK query is rejected and we never insert that AccountId into state
-          await resolver({
+          const test = await resolver({
             opportunityId,
             opportunityType,
             accountId,
             reduxApi: { dispatch, getState },
           })
 
+          if (typeof test.data === 'object' && Object.keys(test)) {
```

Comment:
Just validating this worked, this won't pass type-check, make it consistent for LP/staking